feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -55,42 +66,65 @@ fun ProfileView(
     )
 }
 
+/**
+ * Function [ProfileContainer] contains the actual view of ProfileView.
+ * It will generate the composable elements to create the user's profile.
+ * @param profileValues - to have the profile values passed from View Model
```

Comment:
Replace **to have the profile values** to **Get profile values**

Reply:
done